samples/qml/view.qml

```diff
@@ -6,6 +6,10 @@ import QtQuick 2.0
 import QtQuick.Controls 2.0
 import QtQuick.LocalStorage 2.0
 
+// Even though GleanInstance will only be used inside metrics.js,
```

Comment:
And I bet this will be the way it will have to be used anyway :-)

Reply:
Yeap, that was my though. I worry about about how are things going to be done when there in more than one view, will we re-instantiate Glean for each view everytime? not really sure.